kotlin-gradle/src/main/kotlin/EmitLog.kt

```diff
@@ -0,0 +1,45 @@
+import com.rabbitmq.client.BuiltinExchangeType
+import com.rabbitmq.client.ConnectionFactory
+
+
+/**
+ * Author: vincent
```

Comment:
We don't have such headers in other versions. Git and GitHub will preserve original authorship information. May I ask you to delete these headers?

Reply:
of course